README.md

```diff
@@ -1,6 +1,10 @@
 # bert-cosine-sim
 Fine-tune BERT to generate sentence embedding for cosine similarity. Most of the code is copied from [huggingface's bert project](https://github.com/huggingface/pytorch-pretrained-BERT).
 
+# Preparing data for training
```

Comment:
`Preparing data for training`  --> `Download data and pre-trained model for fine-tuning`